web/modules/custom/joinup_invite/src/InvitationMessageHelper.php

```diff
@@ -93,8 +93,14 @@ protected function getDefaultArguments(InvitationInterface $invitation): array {
       $arguments["@invitation:${action}_url"] = Url::fromRoute('joinup_invite.update_invitation', $url_arguments, $url_options)->toString();
     }
 
-    $arguments['@invitation:role'] = $invitation->field_invitation_og_role->isEmpty() ? '' :
-      strtolower($invitation->field_invitation_og_role->first()->entity->label());
+    $arguments['@invitation:role'] = '';
+    if ($invitation->get('field_invitation_og_role')->isEmpty()) {
+      return $arguments;
+    }
```

Comment:
We should check that the field exists before calling `isEmpty()` on it, the `discussion` bundle doesn't have this field so if this method is called using a discussion bundle this will throw a fatal error.